doc/man7/RSA-PSS.pod

```diff
@@ -30,15 +30,19 @@ By default no parameter restrictions are placed on the generated key.
 
 =head1 NOTES
 
-The public key format is documented in RFC4055.
+The public key format is documented in RFC 8017, which differs from RFC 4055
+L<section 3.1|https://www.rfc-editor.org/rfc/rfc4055.html#section-3.1>
+by outputting two extra NULL values (for optional
```

Comment:
The reason I suggested changing this to 'including' previously is that I dislike 'output' here. It feels imprecise: the format doesn't output anything.